js/settingsHelpers.js

```diff
@@ -412,14 +590,19 @@ export const changeEmail = async (email, userData) => {
  *   the same is true for homePhone and otherPhone (canWeVoicemailHome and canWeVoicemailOther)
  *   if user deletes a number, set canWeVoicemail and canWeText to '' (empty string) --per spec on 05-09-2023
  *   if user updates a number, ensure the canWeVoicemail and canWeText values are set
+ *   Update: 05-26-2023 do not include email addresses in user profile archiving. Exclude those keys from the history object.
 */
 const findChangedUserDataValues = (newUserData, existingUserData, type) => {
   const changedUserDataForProfile = {};
   const changedUserDataForHistory = {};
+  const excludeHistoryKeys = [cId.prefEmail, cId.additionalEmail1, cId.additionalEmail2, cId.firebaseAuthEmail];
+
   Object.keys(newUserData).forEach(key => {
     if (newUserData[key] !== existingUserData[key]) {
       changedUserDataForProfile[key] = newUserData[key] ?? '';
-      changedUserDataForHistory[key] = existingUserData[key] ?? '';
+      if (!excludeHistoryKeys.includes(key)) {
+        changedUserDataForHistory[key] = existingUserData[key] ?? '';
```

Comment:
`existingUserData[key] ?? ''` could create value inconsistency between `existingUserData` and `changedUserDataForHistory` on a key.

Reply:
If key is not in `existingUserData`, there is no need to update. Fallback to "" could be an issue.